src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = sourceString
+    .split(';')
```

Comment:
There's no need to make a copy of the array before operations like `map` or `filter` as they do not mutate the original array.